src/pages/index.tsx

```diff
@@ -21,16 +31,94 @@ import tClientNext from "../tClientNext";
 import PublicUser from '../shared/publicModels/PublicUser';
 import PublicGroup from '../shared/publicModels/PublicGroup';
 import { MdVideocam } from 'react-icons/md';
+import { HSeparator } from 'horizon-ui/components/separator/Separator';
+import { IYuanjianUser } from 'shared/user';
+import { toast } from "react-toastify";
 
 const AppIndex: NextPageWithLayout = () => {
   const user = useUserInfo();
-  return <Box paddingTop={'80px'}><Meetings /></Box>;
+  // if (!user.name) {
+  //   return <Box paddingTop={'80px'}><SetNameModal u={user} /><Meetings /></Box>
+  // };
+  return <Box paddingTop={'80px'}> {user.name ? <></>:<SetNameModal />}<Meetings /></Box>
 }
 
 AppIndex.getLayout = (page) => <AppLayout>{page}</AppLayout>;
 
 export default AppIndex;
 
+function SetNameModal() {
+  const brandStars = useColorModeValue('brand.500', 'brand.400');
+  const [isOpen, setOpen] = useState(true);
+  const [name, setName] = useState('');
+
+  const handleSubmit = async () => {
+    if (name) {
+      const updatedUser = {
+        name: name,
+      };
+
+      tClientBrowser.user.updateProfile.mutate(updatedUser).then(
+        res => {
+          if (res === "ok") {
+            console.log("user name update succeeded");
+            setOpen(false);
+          }
+        }
+      ).catch(e => toast.error(e.message, { autoClose: false }))
+    };
+  };
+
+  return (
+    <Modal isOpen={isOpen} onClose={() => undefined}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>新用户登录</ModalHeader>
+        <ModalBody>
+          <Text>
+            请填写中文全名
+          </Text>
+          <Box mt={4}>
+            <Flex align='center' mb='25px'>
+              <HSeparator />
+            </Flex>
+            <FormControl>
+              {!name && (
+                <Alert status="error" mt={4}>
+                  <AlertIcon />
+                  姓名不能为空
+                </Alert>
+              )}
+              <FormLabel display='flex'>
+                姓名<Text color={brandStars}>*</Text>
+              </FormLabel>
+              <Input
```

Comment:
remove as many styling props from both <Input> and <Button>. do we really need them? Right now we don't need to (and shouldn't) fine tune local styles. We should adjust styling globally.